src/vhost_user/message.rs

```diff
@@ -659,8 +659,6 @@ impl VhostUserMsgValidator for VhostUserConfig {
     fn is_valid(&self) -> bool {
         if (self.flags & !VhostUserConfigFlags::all().bits()) != 0 {
             return false;
-        } else if self.offset < 0x100 {
-            return false;
         } else if self.size == 0
             || self.size > VHOST_USER_CONFIG_SIZE
             || self.size + self.offset > VHOST_USER_CONFIG_SIZE
```

Comment:
Seems we should also check for "self.offset >= VHOST_USER_CONFIG_SIZE", otherwise "self.size + self.offset" may wrap around.

Reply:
Good catch, you're correct. Switched to checked_add and added a test.